testing/src/test/scala/compstak/kafkastreams4s/tests/ShapelessTests.scala

```diff
@@ -0,0 +1,94 @@
+package compstak.kafkastreams4s
+
+import java.sql.Timestamp
+
+import _root_.shapeless._, labelled._, record._
+
+import shapeless._
+
+import compstak.circe.debezium.{DebeziumFieldSchema, DebeziumSchemaPrimitive}
+import compstak.kafkastreams4s.debezium.{DebeziumCompositeType, DebeziumPrimitiveType}
+
+class ShapelessTests extends munit.FunSuite {
+  implicit val tsDP = new DebeziumPrimitiveType[Timestamp] {
+    def debeziumType: DebeziumSchemaPrimitive = DebeziumSchemaPrimitive.Int64
+  }
+
+  type CompProposal =
+    Record.`Symbol("id") -> Long, Symbol("proposalsIdx") -> Option[Long], Symbol("compBatchId") -> Option[Long], Symbol("dateCreated") -> Timestamp, Symbol("dateUpdated") -> Timestamp, Symbol("marketId") -> Option[Long], Symbol("validated") -> Boolean, Symbol("compVersionId") -> Option[Long], Symbol("verifiedBy") -> Option[String], Symbol("monthly") -> Option[Boolean], Symbol("rollbackReason") -> Option[String], Symbol("stashed") -> Boolean, Symbol("stashReason") -> Option[String]`.T
```

Comment:
I guess there's no way to break the line here somewhere?

Reply:
Not with the embedded expression trick. I could avoid the trick, I think, if we really insisted on it.